worker/categorize.py

```diff
@@ -395,7 +395,7 @@ def _is_viable_candidate(self, phrase):
         words = phrase.split()
 
         # Ignore stop words
-        if words[0].lower() in STOP_WORDS:
+        if len(words) == 0 or words[0].lower() in STOP_WORDS:
```

Comment:
Added this because I got this stack trace:

INFO:sqlalchemy.engine.base.Engine:(545,)
Traceback (most recent call last):
  File "manage.py", line 140, in <module>
    manager.run()
  File "/Users/scottgladstone/Sourcemash/flask/lib/python2.7/site-packages/flask_script/__init__.py", line 412, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/Users/scottgladstone/Sourcemash/flask/lib/python2.7/site-packages/flask_script/__init__.py", line 383, in handle
    res = handle(*args, **config)
  File "/Users/scottgladstone/Sourcemash/flask/lib/python2.7/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
  File "manage.py", line 89, in feed_seed
    scrape_and_categorize_articles()
  File "/Users/scottgladstone/Sourcemash/worker/scraper.py", line 40, in scrape_and_categorize_articles
    categories = categorizer.categorize_item(item.title, text_only)
  File "/Users/scottgladstone/Sourcemash/worker/categorize.py", line 93, in categorize_item
    keyword_candidates = self._get_keyword_candidates(title, text)
  File "/Users/scottgladstone/Sourcemash/worker/categorize.py", line 116, in _get_keyword_candidates
    text_ngrams = self._get_valid_ngrams(text)
  File "/Users/scottgladstone/Sourcemash/worker/categorize.py", line 143, in _get_valid_ngrams
    if self._is_viable_candidate(ngram):
  File "/Users/scottgladstone/Sourcemash/worker/categorize.py", line 398, in _is_viable_candidate
    if words[0].lower() in STOP_WORDS:
IndexError: list index out of range